.editorconfig

```diff
@@ -0,0 +1,23 @@
+root = true
+
+[*]
+charset=utf-8
+end_of_line=lf
+insert_final_newline=false
+indent_style=space
+indent_size=4
+trim_trailing_whitespace=true
+
+[*.md]
+trim_trailing_whitespace=false
+
+[*.json]
+trim_trailing_whitespace=true
+indent_style=space
+indent_size=4
+
+[{*.yml,*.yaml}]
+max_line_length = 100
+trim_trailing_whitespace=true
+indent_style=space
+indent_size=2
```

Comment:
kinda still like indent_size 4 here.

Reply:
Does that work? I did not know that Yaml supported that. I agree that it would be easier to read with 4 spaces, if that works. All tough all our current specifications uses 2 spaces, so we would have to change them if we want them to comply with out editorconfig.